docker/lr-winnowmap/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:18.04
+
+RUN apt-get -qqy update && \
+    apt-get -qqy dist-upgrade && \
+    apt-get install -y make gcc g++ libz-dev wget xz-utils git
+
+
+RUN wget https://github.com/marbl/meryl/releases/download/v1.3/meryl-1.3.Linux-amd64.tar.xz && \
```

Comment:
There are several optimizations (or different balances) you can make here.
Let's talk in Zoom what they are.